va/va.h

```diff
@@ -4355,6 +4355,37 @@ VAStatus vaDeriveImage (
     VAImage *image	/* out */
 );
 
+/**
+ * Device Infomation *
+ */
+typedef enum {
+    /** \brief modifier support, should be several Modifier types */
+    VADeviceAttribModifier = 1,
+    /** \brief memory type support , on chip memory etc.*/
+    VADeviceAttribMemoryType    = 2
+}VADeviceAttribType;
```

Comment:
We already had #387 which proposed device attributes in a little bit different form. We considered to expose the following info in this PR:
* vendor_id
* device_id
* has_local_memory

Can you, please, add vendor and device ID attributes?